onnx/common/tensor.h

```diff
@@ -139,6 +139,221 @@ struct Tensor final {
     has_name_ = true;
     name_ = std::move(name);
   }
+
+  bool is_raw_data() const {
+    return is_raw_data_;
+  }
+
+  template<typename T, int64_t block_size>
+  void bin_func(void (*f)(void*, const void*), Tensor& a, std::vector<T>& T_data_, std::vector<T>& a_T_data_) {
```

Comment:
nit: the naming of the parameters are not very intuitive.

Reply:
changed void to T and moved template to private as the only purpose is to serve apply_binary_function